arrays.js

```diff
@@ -26,16 +35,20 @@ function findFirstStringStartingWithLetter(letter, strings) {
 **************************************************************/
 function isPresentIncluded(presentName, presents) {
   //TODO: Add your code here
+  return presents.map(function (present) {
+    return present.includes(presentName, 0);
```

Comment:
Well, same here. You had to lowercase both 'presentName' and 'presents' to avoid upper and lower case situations. Additionally, if you want to start checking from the first index, you don't have to add the second argument (0). 
Finally, you could apply includes directly to the array.